pkg/rtc/mediatrackreceiver.go

```diff
@@ -684,9 +684,12 @@ func (t *MediaTrackReceiver) UpdateAudioTrack(update *livekit.UpdateLocalAudioTr
 			t.trackInfo.DisableDtx = true
 		}
 	}
+	ti := t.trackInfoCloneLocked()
 	t.lock.Unlock()
 
 	t.updateTrackInfoOfReceivers()
+
+	t.params.Telemetry.TrackPublishedUpdate(context.Background(), t.PublisherID(), ti)
```

Comment:
I'm wondering if we can send only if it's changed? in case of developer mis-use, that causes us to insert a bunch of DB records.

Reply:
Yeah, will look at checking for chages before sending.

Reply:
Addressed in this [commit](https://github.com/livekit/livekit/pull/2723/commits/faee06b7846a1a0752b1ae39fe1f50d8598a1915).